docs/es_config.txt

```diff
@@ -0,0 +1,349 @@
+facilmente modificable tanto desde línea de comandos como editando los archivos de configuración directamente.
+Todos los archivos de configuración de UCI se encuentran en el directorio `/etc/config/`.
+Los archivos de configuración de LibreMesh que pueden ser editados para son: `/etc/config/lime-node` y
+`/etc/config/lime-community`.
+El resto de los archivos de configuración no deben ser modificados.
+
+== Niveles de configuración *_Comunidad_* y *_Nodo_*
+
+Para ayudar a la comunidad a mantener sus configuraciones, la configuración de los nodos de la red
+se estructura jerarquicamente en 3 niveles:
+
+1. *_Nodo_*, archivo `/etc/config/lime-node`
+2. *_Comunidad_*, archivo `/etc/config/lime-community`
+3. *_Defecto_*, archivo `/etc/config/lime-defaults`
+
+Donde la configuración realizada a nivel de *_Nodo_* toma prioridad por sobre *_Comunidad_* y esta  a su
+vez tiene prioridad por sobre el *_Defecto_*.
+De esta forma cuando la mayoria de los nodos comparten cierta caracteristica la misma conviene
+configurarse a nivel comunidad y solo modificarla en el nivel de *_Nodo_* en los nodos donde esa configuración difiere.
+El nivel *_Default_* contiene todos los valores de configuración en su valor por defecto.
+
+=== Ejemplo
+
+Supongamos que queremos configurar la distancia máxima wireless de los nodos de 2.4GHz de la comunidad
+en 300 metros pero en dos de los nodos A y B configurarla en 1000m. Lo podríamos hacer de la siguiente forma:
+
+En `lime-community` de todos los nodos tendriamos:
+
+```python
+config lime 'wifi'
+	# other options
+	option distance_2ghz '300'
+```
+
+Y en los archivos `lime-node` de los nodos A y B:
+
+```python
+config lime 'wifi'
+	option distance_2ghz '1000'
+```
+
+=== Funcionamiento
+
+Al correr el comando `lime-config` lo primero que sucede es que se procesan los niveles *_Nodo_*,
+*_Comunidad_* y *_Default_* y se genera el archivo `/etc/config/lime-autogen`.
+Este archivo es el que contiene la configuración que realmente va a utilizarse por libremesh para configurar el sistema.
+
+
+The executable `lime-config` reads such file and writes all the resulting complex configuration in many other configuration files.
+In order to apply modifications to the configuration file, it has to be executed running `lime-config` in the
+node's root console and the router has to be rebooted with the `reboot` command.
+
+
+=== Lista de secciones y opciones
+
+La configuración se divide en tres secciones principales: `system`, `network` y `wifi`.
+
+Para tener información detallada y actualizada de todas las opciones ver la referencia actualizada
+en {lime-example}.
+
+.Opciones de sistema
+
+Las siguientes opciones de la seccion `system` son las más relevantes:
+----
+config lime system
+	option hostname 'LiMe-%M4%M5%M6'
+	option domain 'thisnode.info'
+----
+
+Te sugerimos que cambies el dominio +thisnode.info+ por el sitio de tu comunidad, por ejemplo +option domain 'altermundi.net'+.
```

Comment:
This contradicts the suggestion in en_config. Which states:
We warmly suggest to change the domain from lan to a new subdomain of your community’s website, for example option domain mesh.altermundi.net. Beware that this domain (mesh.altermundi.net) should not already exist out of the network (i.e. is bad idea to use altermundi.net).

Reply:
Right, by mistake I translated an older version of the en_config docs